src/platform/forms-system/src/js/widgets/SelectWidget.jsx

```diff
@@ -28,14 +28,20 @@ function SelectWidget({
   onBlur,
   placeholder,
 }) {
-  const { enumOptions, labels = {} } = options;
+  const {
+    enumOptions,
+    labels = {},
+    widgetProps = {},
+    selectedProps = {},
+  } = options;
+
   return (
     <select
       id={id}
       name={id}
       multiple={multiple}
-      className={`${options.widgetClassNames} ${
-        disabled ? 'vads-u-background-color--gray-lighter' : ''
+      className={`${options.widgetClassNames || ''}${
+        disabled ? ' vads-u-background-color--gray-lighter' : ''
```

Comment:
Is this related to the `aria-describedby` changes?

Reply:
No, but it was added an "undefined" class name to the select.